modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -291,6 +294,49 @@ export default class AdvancedOpts {
             .text( d => d.label );
 
         fieldLabel.merge(fieldLabelEnter);
+
+        let fieldLabelButton = fieldLabelWrap.selectAll( '.hoot-field-label-button' )
+            .data( [d] );
+
+        fieldLabelButton.exit().remove();
+
+        let fieldButtonEnter = fieldLabelButton.enter()
+            .append('button')
+            .classed('hoot-field-label-button', true )
+            .call(svgIcon('#iD-icon-inspect', 'adv-opt-icon', ''))
+                 .on('click', function () {
+                        d3.event.stopPropagation();
+                        d3.event.preventDefault();
+
+                        let getDescId = d3.select(`#${d.id}`);
+
+                        if (this.showing && d3.select('.adv-opt-reference').datum().id  === d.id ) {
+                            if (d.input === 'checkbox') {
+                                getDescId.classed('hoot-form-field-checkbox', true);
+                                getDescId.classed('hoot-form-field-checkbox-clicked', false);
+                            }
+
+                            d3.select('.adv-opt-reference').remove();
+                            this.showing = false;
+                        }
+                        else {
+                            if (d.input === 'checkbox') {
+                                getDescId.classed('hoot-form-field-checkbox', false);
+                                getDescId.classed('hoot-form-field-checkbox-clicked', true );
+
+                            }
+                                getDescId
+                                    .append('div')
```

Comment:
![image](https://user-images.githubusercontent.com/15100429/73457604-e6a91f80-4341-11ea-9a4d-df94dade49cd.png)

There seems to be a state in which having 2 dropdowns open leads to adding divs over and over if you click on one of the info buttons.

I think we can avoid this if we don't append on click, but show a div that we add in the same way we do this button  (d3 data binding cycle).

More succinctly I think you...

1. add the description div not in the callback, but right underneath were 339 following d3 data binding flow
2. add a class for hiddenn to the div (there should be one we have already in the ui), if not, a display = none
3. then in the callback, all you do is set the value of the class/display property that shows/hides the div as the inverse of what it is `selection.classed('name-of-class, !selection.classed('name-of-class')`